frontend/src/container/ListOfDashboard/utils.ts

```diff
@@ -0,0 +1,18 @@
+export const filterDashboard = (
+	searchValue: string,
+	dashboardList: any,
```

Comment:
`any` can be replaced with `Dashboard[]` can be imported from 

`import { Dashboard } from 'types/api/dashboard/getAll';`